status-im/status-react
# [FIX] remove message cache

message cache was initialized when logging-in so it was basically useless
it was causing a bug when switching account and getting a contact request
from the same user in 2 accounts without killing the app because the cache
was not reset and the subsequent contact requests were ignored

# to test (try first on develop)
- create account A and B on same device
- invite A and B with account C
- check A first, C username should be visible
- check B, C still has random name on develop, should show username on this branch

status: ready
